apps/admin-gui/src/assets/i18n/en.json

```
@@ -2019,6 +2023,26 @@
       "REMOVE_SPONSOR_DISABLED": "You are not authorized to remove sponsor",
       "SUCCESS": "Sponsors successfully edited"
     },
+    "BULK_INVITE_MEMBERS": {
+      "TITLE": "Invite multiple members",
+      "INSERT_HERE": "Here insert correctly formatted text",
+      "DESCRIPTION_VO": "The persons you are inviting will receive an email with a link to organizations application form. You can set up an email template in: (Organization) Settings -> Notifications.",
```

```suggestion
      "DESCRIPTION_VO": "The persons you are inviting will receive an email with a link to organization's application form. You can set up an email template in: (Organization) Settings -> Notifications.",
```
same with "group's" below